src/passes/ExtractFunction.cpp

```diff
@@ -43,7 +43,7 @@ struct ExtractFunction : public Pass {
     }
     if (!found) {
       std::cerr << "could not find the function to extract\n";
-      abort();
+      exit(1);
```

Comment:
These two lines could be `Fatal() << "..message.."` - we should have been more consistent with that before, but may as well fix it now.